### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#191] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-20 224001](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8a895ad1-5a8c-4b1e-b8e2-b0c50e37bfe6)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione Confermata in Classe
**(1)** Le distribuzioni marginali sono:

$$\mathbb{P}(X=x)=\binom{3}{x}\binom{4}{3-x}/\binom{7}{3},\ x=0,1,2,3$$

$$\mathbb{P}(Y=y)=\binom{2}{y}\binom{5}{3-y}/\binom{7}{3},\ y=0,1,2$$

Mentre quella congiunta:

$$\mathbb{P}(X=x,Y=y)=\binom{3}{x}\binom{2}{y}\binom{2}{3-(x+y)}/\binom{7}{3},\ (x+y)=1,2,3$$

**(2)** Per definizione $cov(X,Y)=\mathbb{E}(XY)-\mathbb{E}(X)\mathbb{E}(Y)$:

$$\mathbb{E}(XY)=\sum^3_{x=0}\sum^2_{y=0}(xy)\cdot \mathbb{P}(X=x,Y=y)=\frac{(1\cdot 1)\cdot 12}{35}+\frac{(1\cdot 2)\cdot 3}{35}+\frac{(2\cdot 1)\cdot 6}{35}=\frac{6}{7}$$

$$\mathbb{E}(X)=\sum^3_{x=0}x\cdot \mathbb{P}(X=x)=0+\frac{1\cdot 18}{35}+\frac{2\cdot 12}{35}+\frac{3}{35}=\frac{9}{7}$$

$$\mathbb{E}(Y)=\sum^2_{y=0}y\cdot \mathbb{P}(Y=y)=0+\frac{1\cdot 20}{35}+\frac{2\cdot 5}{35}=\frac{6}{7}$$

$$\Rightarrow cov(X,Y)=\frac{6}{7}-\frac{9}{7}\cdot \frac{6}{7}=-\frac{12}{49}$$

Ricordo che $X,Y$ sono indipendenti $\iff \mathbb{P}(X=x)\cdot \mathbb{P}(Y=y)=\mathbb{P}(X=x,Y=y),\forall\ x\in Im(X),\forall \ y\in Im(Y)$.
$X,Y$ non sono indipendenti, infatti (controesempio):

$$\mathbb{P}(X=0)\cdot \mathbb{P}(Y=0)=\binom{4}{3}\cdot \binom{5}{3}/\binom{7}{3}^2\neq 0=\mathbb{P}(X=0,Y=0)$$

**(3)** L'apparecchio funziona se scelte le tre batterie $X+Y=3$, ma calcolare la probabilità della somma di v.a. non indipendenti può essere complicato, allora calcoliamo equivalentemente $\mathbb{P}(X=x,Y=y)$ con il vincolo $x+y=3$, ovvero:

$$\mathbb{P}(apparecchio\ funziona)=\mathbb{P}(X=1,Y=2)+\mathbb{P}(X=2,Y=1)+\mathbb{P}(X=3,Y=0)=\frac{3}{35}+\frac{6}{35}+\frac{1}{35}=\frac{2}{7}$$

**(3 alternativo)** E' possibile anche utilizzare la distribuzione ipergeometrica siccome in questo caso le batterie possono essere distinte tra funzionanti e difettose.

**Comment by dahKidou** [comment]
> **(1)** Le distribuzioni marginali sono: P(X=x)=(3x)(43−x)/(73), x=0,1,2,3
> 
> P(Y=y)=(2y)(53−y)/(73), y=0,1,2
> 
> Mentre quella congiunta: P(X=x,Y=y)=(3x)(2y)(23−(x+y))/(73), (x+y)=1,2,3
> 
> **(2)** Per definizione cov(X,Y)=E(XY)−E(X)E(Y): E(XY)=∑x=03∑y=02(xy)⋅P(X=x,Y=y)=(1⋅1)⋅1235+(1⋅2)⋅335+(2⋅1)⋅635=67
> 
> E(X)=∑x=03x⋅P(X=x)=0+1⋅1835+2⋅1235+335=97
> 
> E(Y)=∑y=02y⋅P(Y=y)=0+1⋅1035+2⋅535=47
> 
> ⇒cov(X,Y)=67−97⋅47=649
> 
> Ricordo che X,Y sono indipendenti ⟺P(X=x)⋅P(Y=y)=P(X=x,Y=y),∀ x∈Im(X),∀ y∈Im(Y). X,Y non sono indipendenti, infatti (controesempio): P(X=0)⋅P(Y=0)=(43)⋅(53)/(73)2≠0=P(X=0,Y=0)
> 
> **(3)** L'apparecchio funziona se scelte le tre batterie X+Y=3, ma calcolare la probabilità della somma di v.a. non indipendenti può essere complicato, allora calcoliamo equivalentemente P(X=x,Y=y) con il vincolo x+y=3, ovvero:
> 
> P(apparecchio funziona)=P(X=1,Y=2)+P(X=2,Y=1)=335+635=935

Nell' ultima richiesta, non manca il caso: P(X = 3, Y = 0), essendo che è possibile che ci siano 3 nuove e 0 usate?

**Comment by LeoCosta02** [comment]
@Elia-Belli la distribuzione marginale e congiunta come fa a venirti cosi?

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/24ca80b9-a07d-4605-b566-ca1c62eebccc)
La definizione scritta cosi é giusta?

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli ma con tabella
![immagine](https://github.com/user-attachments/assets/1fe6b1e5-719c-4e18-a765-00d2286654eb)
